src/server/appserver.h

```diff
@@ -241,6 +244,7 @@ class AppServer : public SharedTools::QtSingleApplication {
         void onCleanup();
         void onRequestReceived(int id, RequestNum num, const QByteArray &params);
         void onRestartClientReceived();
+		void onMessageReceivedFromAnotherProcess(const QString &message, QObject *);
```

Comment:
Indentation.

Reply:
Strange, I do not see it in my IDE...
![Screenshot 2024-04-17 084235](https://github.com/Infomaniak/desktop-kDrive/assets/23190613/c7672986-9ecc-4724-89dc-704ec2d57981)

Reply:
No, I think it was invisible characters on my side. Sorry.
Should be ok now

Reply:
> No, I think it was invisible characters on my side. Sorry. Should be ok now
I encountered this issue before, It is  from using tabs instead of four spaces.